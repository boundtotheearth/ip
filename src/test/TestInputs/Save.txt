todo read book
event go home /at 2020-10-10
deadline return book /by 2020-10-10
