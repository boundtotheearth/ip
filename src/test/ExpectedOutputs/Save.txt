TODO|false|read book
EVENT|false|go home|2020-10-10
DEADLINE|false|return book|2020-10-10
